### PR TITLE
Adding persons and ticketing flows

### PR DESCRIPTION
**Pull request by ividamileen**

### CODE REVIEW

File: src/1 Sysops Squad/system.puml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@startuml
+!include https://raw.githubusercontent.com/adrianvlupu/C4-PlantUML/latest/C4_Container.puml
+
+LAYOUT_WITH_LEGEND()
+
+Person(c, "Customer", "A customer of  Penultimate Electronics.")
```
Comment (mattjonesorg) [inline src/1 Sysops Squad/system.puml]: As discussed, we want to use more clarifying variable names.

Comment (mqmurtuza) [reply src/1 Sysops Squad/system.puml]: I accept this and it looks great

**Review by mqmurtuza — APPROVED** [review]
I approve it

File: src/1 Sysops Squad/system.puml (hunk 1 above)
Comment (ividamileen) [reply src/1 Sysops Squad/system.puml]: DOne, updated variable names

**Pull request merged**